### PR TITLE
Development_Setup.rst: Change the text and URL

### PR DESCRIPTION
**Pull request by amritpals**
Change the text syntax and URL for newcomers guide.

Fixes https://github.com/coala/coala/issues/3643

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

**After you submit your pull request, DO NOT click the 'Update Branch' button**

### CODE REVIEW

**Comment by amritpals** [comment]
@ankit01ojha I was trying it but few things messed up that I couldn't edit existing commit.

File: docs/Developers/Development_Setup.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Repositories
 ------------
 
 If you are interested in contributing to coala, we recommend that you read
-our :ref:`newcomers' guide <newcomer-guide>` to familiarize yourself with our
+our `newcomers guide <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`__ to familiarize yourself with our
```
Comment (nemaniarjun) [inline docs/Developers/Development_Setup.rst]: You can use `coala.io/newcomer`instead of the whole link :)

**Comment by incorrectusername** [comment]
change `Fixes` to `Closes`.

File: docs/Developers/Development_Setup.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ Repositories
 ------------
 
 If you are interested in contributing to coala, we recommend that you read
-our :ref:`newcomers' guide <newcomer-guide>` to familiarize yourself with our
+our `newcomers guide <http://coala.io/newcomer>`__ to familiarize yourself with our
```
Comment (rhemon) [inline docs/Developers/Development_Setup.rst]: why  remove apostrophe after `newcomers`?

**Comment by amritpals** [comment]
I will update this pull request by today EOD.

**Comment by incorrectusername** [comment]
ack a723f087183f7bd6338d3b9c2233374c9668c22d

**Comment by Asnelchristian** [comment]
The commit shortlog isn't quite clear, (we have no clue about which text you are talking about )

**Comment by Asnelchristian** [comment]
what do you think about ``Modify newcomers' guide URL``

**Comment by Asnelchristian** [comment]
Hi,

It looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

We also have a quick video tutorial on how to rebase: [https://asciinema.org/a/78683](https://asciinema.org/a/78683). That should help you understand the basics of how it works and what you should be doing.

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

(Be sure to use the coala-bears URL if you're on coala bears!)

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by RJ722** [comment]
@amritpals still working on it?

**Comment by jayvdb** [comment]
Already fixed.

**Pull request closed without merging**